### ISSUE TITLE
Team Settings and User Settings

### ISSUE THREAD

**Issue by hesantan** [axodox/AxoTools#91]
I think it would be awesome if basic settings could be declared globally for everyone working with the same repository and are part of version control. There could be a **settings.user.json** for individual user settings that are not part of version control and can be adjusted to your individual needs. It could be a little complicated to handle the syncing between the two. Let me know what you think and it's worth it and/or feasible.

**Comment by axodox** [comment]
Yes, I think this is feasible, it could work in a way, that it would take the global settings, unless you override them. However I pretty busy right now with my daytime work... So this might have to wait a little, as I have some more pressing issues to work on in AxoCover. Still if you have time to it - I think it is doable in a few hours, you can do it and make a pull request.
